cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -81,7 +82,7 @@ def test_memory_checkbox(small_test_vm, provider, soft_assert):
     # Make sure the VM is powered on
     small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_ON, cancel=False)
     # Try to create snapshot with memory on powered on VM
-    has_name = provider.one_of(RHEVMProvider)
+    has_name = not provider.one_of(RHEVMProvider)
```

Comment:
**QUESTION:** This inversion is because it was bad to begin with?

Reply:
Correct.